### PR TITLE
nginx for rsiviz

### PR DESCRIPTION
**Pull request by e-carlin**

### CODE REVIEW

**Review by robnagler — CHANGES_REQUESTED** [review]

File: rsconf/package_data/rsiviz/nginx.conf.jinja [hunk 1]
```diff
@@ -0,0 +1,18 @@
+client_max_body_size 200m;
+
+server {
+    listen {{ nginx.index_port }};
+    root {{ nginx.default_root }};
+    server_name {{ nginx.vhost }};
+    location /kpmcyjtsoxks/ {
```
[the remark this answers is not in the document]
Comment (e-carlin) [reply rsconf/package_data/rsiviz/nginx.conf.jinja]: doh, thanks b04db4b

File: rsconf/package_data/rsiviz/nginx.conf.jinja [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    }
+}
+server {
+    listen {{ nginx.flask_port }};
+    root {{ nginx.default_root }};
+    server_name {{ nginx.vhost }};
+    location /kpmcyjtsoxks/ {
+        proxy_pass http://127.0.0.1:{{ nginx.docker_flask_port }}/;
+    }
+}
```
[the remark this answers is not in the document]
Comment (e-carlin) [reply rsconf/package_data/rsiviz/nginx.conf.jinja]: b04db4b

**Comment by e-carlin** [comment]
Merging in order to keep moving but please review and i will fix up

File: rsconf/package_data/rsiviz/nginx.conf.jinja [hunk 3]
```diff
@@ -0,0 +1,18 @@
+client_max_body_size 200m;
```
Comment (robnagler) [inline rsconf/package_data/rsiviz/nginx.conf.jinja:1]: Probably needs to be larger. May want it configurable.

File: rsconf/package_data/rsiviz/nginx.conf.jinja [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+server {
+    listen {{ nginx.index_port }};
+    root {{ nginx.default_root }};
+    server_name {{ nginx.vhost }};
```
Comment (robnagler) [inline rsconf/package_data/rsiviz/nginx.conf.jinja:6]: Terminate with TLS (see sirepo). You can use a *.radiasoft.org service name.

File: rsconf/package_data/rsiviz/nginx.conf.jinja [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+    location /{{ rsiviz.url_secret }}/ {
```
Comment (robnagler) [inline rsconf/package_data/rsiviz/nginx.conf.jinja:7]: You can remove this, and instead have a secret service host name. It looks from the proxy_pass that there's no API's except "/"?

**Pull request merged**